sdk/confidentialledger/azure-security-confidentialledger/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
+## 1.0.0 (2022-07-19)
+- GA ("Generally Available") release for 2021-07-19 service version. 
```

Comment:
The service version should be what's defined in `ConfidentialLedgerServiceVersion`.
```suggestion
- GA ("Generally Available") release for 2022-05-13 service version. 
```